core/src/main/java/de/danoeh/antennapod/core/feed/Feed.java

```diff
@@ -390,12 +393,59 @@ public void setFeedIdentifier(String feedIdentifier) {
         this.feedIdentifier = feedIdentifier;
     }
 
-    public String getPaymentLink() {
-        return paymentLink;
+    public ArrayList<FeedFunding> extractPaymentLinks(String payLinks) {
+        if (StringUtils.isBlank(payLinks)) {
+            return null;
+        }
+        // old format before we started storing the urls as pay= and fund=
```

Comment:
Is this comment still valid?

Reply:
fixed the comment, we still want to note that the storage is different before the podcastindex funding tag parsing was added, so the database would be a little different